features/unattached_commands.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Command behaviour when unattached
+
+    Scenario: Unattached detach in a trusty lxd container
+        Given a trusty lxd container
+          And ubuntu-advantage-client is installed
+        When I run `ua detach`
+        Then I will see the following on stderr:
+            """
+            This machine is not attached to a UA subscription.
+            See https://ubuntu.com/advantage
+            """
+
+    Scenario: Unattached refresh in a trusty lxd container
+        Given a trusty lxd container
+          And ubuntu-advantage-client is installed
+        When I run `ua refresh`
+        Then I will see the following on stderr:
```

Comment:
why the weird indentation between given/and and when/then?

Reply:
`Given`/`When`/`Then` all align at the same column, `And` aligns its last character with the last character of the line it attaches to.

More than happy to standardise on something else, but this isn't _completely_ arbitrary. ^_^

Reply:
ah! sorry, I was looking at it from the end of the line... not sure why. Ignore me

Reply:
No worries! It's more obvious what the pattern is when there's more than one "And", for sure.